### DISCUSSION TITLE
Asus ProArt PA32 series...

### DISCUSSION THREAD

**Discussion by srgzx** [MonitorControl/MonitorControl#1031] [Q&A] [answered]
Doesn't seem to work with this app.  Any possible way to get it working?

<img width="697" alt="Screen Shot 2022-05-08 at 11 59 21 AM" src="https://user-images.githubusercontent.com/103330345/167304552-eb7056a7-87bf-4af5-959a-5f965a92a93b.png">

I'm using an M1 Mac Mini and connecting this PA32UCX-PK monitor via Thunderbolt

**Comment by waydabber** [comment, marked as answer]
Wow, that is an outdated version there. :)

**Reply by waydabber** [reply]
Try downloading this one: https://github.com/MonitorControl/MonitorControl/releases/tag/v4.0.2

**Reply by srgzx** [reply]
Oh my god.  It works!  Thank you.  I'm an idiot, apparently.  I actually had 4.0.2 in my downloads folder, but I never installed it, and was still running the older one.
